benchopt/utils/r_helpers.py

```diff
@@ -1,27 +1,30 @@
 import os
 
-# Make sure that R_HOME is loaded from the current interpreter to avoid
-# using the parent interpreter R_HOME in the sub-interpreter.
+# # Make sure that R_HOME is loaded from the current interpreter to avoid
+# # using the parent interpreter R_HOME in the sub-interpreter.
```

Comment:
```suggestion
# Make sure that R_HOME is loaded from the current interpreter to avoid
# using the parent interpreter R_HOME in the sub-interpreter.
```